tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -381,6 +381,14 @@ function reset_runtime() {
 }
 
 function main() {
+	echo "Environment variables passed to this script"
+	echo "* NODE_NAME: ${NODE_NAME}"
+	echo "* DEBUG: ${DEBUG}"
```

Comment:
Running with `set -o nounset` and `DEBUG` can be unset (because in some `if`s you set a default value `${DEBUG:-"false"}`). Don't know if the other variables are mandatory.

Reply:
Fixed, thanks!